notify/amqp_setup.py

```diff
@@ -0,0 +1,123 @@
+import os 
+import pika
+from os import getenv
+
+RABBITMQ_HOSTNAME = getenv("RABBITMQ_HOSTNAME") or "13.229.231.31"
+RABBITMQ_PORT = getenv("RABBITMQ_PORT") or 5672
+RABBITMQ_USERNAME = getenv("RABBITMQ_USERNAME") or "guest"
+RABBITMQ_PASSWORD = getenv("RABBITMQ_PASSWORD") or "guest"
+EXCHANGE = "topic_exchange"
+EXCHANGE_TYPE = "topic"
+MS_BASE_URL = "http://localhost:5001"
+
+PROJECT_CREATE_QUEUE = "project_create"
+PROJECT_MILESTONES_REWARD_QUEUE = "project_milestones_reward"
+PROJECT_MILESTONES_PENALISE_QUEUE = "project_milestones_penalise"
+PROJECT_MILESTONES_VERIFY_QUEUE = "project_milestones_verify"
+PROJECT_MILESTONES_UPDATE_QUEUE = "project_milestone_update"
+BUY_PROJECTS_PAYMENT_SUCCESS_QUEUE = "buy_projects_payment_success"
+BUY_PROJECTS_PAYMENT_FAILED_QUEUE = "buy_projects_payment_failed"
+BUY_PROJECTS_NOTIFY_PAYMENT_FAILED_QUEUE = "buy_projects_notify_payment_failed"
+UPCOMING_MILESTONE_PROJECT_POLICE_QUEUE = "upcoming_milestone_project_police"
```

Comment:
let's follow the convention of naming the queues, following a similar structure as the `events.<from>.<to>.<resource>.<action>` key binds of the rabbitmq, in this format: `<queue>_<action>`